README.md

```diff
@@ -12,16 +12,54 @@ This service implements:
 
 Tests for the service can be run with ``npm test``. 
 
-Getting Started
+Key Server Setup and Deployment
 ====================
-1. Git clone this repository.
-2. Do an ``npm install`` in the root folder.
-3. Run ``node bin/server.js`` to start the service.
-4. Import public keys with ``node bin/admin.js import <path to CSV file containing xpubs>``
-5. The service should be accessible in the browser via ``http://localhost:6833/key`` 
-6. Start obtaining a key by issuing a curl command like:
+The key server is BitGo's interface to the KRS service. It is responsible for storing, maintaining, and distributing public keys.
 
-``curl -H "Content-Type: application/json" -d '{ "customerId": "acme-inc-123", "coin": "btc", "userEmail": "user@example.com", "custom": { "phone": "1-800-123-4567" } }' http://localhost:6833/key``
+**Under no circumstance should private keys be stored on the key server.**
+
+1. The KRS server is developed in NodeJS. Install NodeJS and NPM
+
+    [NodeJS Installation Guide](http://howtonode.org/how-to-install-nodejs)
+2. The KRS server stores public keys in a MongoDB database. Install MongoDB on the same server as the KRS server, or on a separate server on the local network.
+
+    [MongoDB Installation Guide](https://docs.mongodb.com/manual/installation/) 
+3. Clone this repository
+4. Run `npm install` in the KRS repository folder to install required libraries for the KRS server.
+5. Configure the KRS service (see Configuration below)
+6. Start the server with `npm start`
+7. The service will be available via `http://localhost:6833/key`
+8. Obtain a key by issuing a curl command like:
+
+`curl -h "Content-Type: application/json" -d "{ "customerId": "123abc", "coin: "btc", "userEmail": "user@example.com", "custom": { } }" http://localhost:6833/key`
```

Comment:
Is this a good place to define the params (required yes/no, what they mean)? Otherwise a user would have to read the code or trial/error or use the BitGo UI

Reply:
This is only an example to test that the server is working correctly - KRS providers will never need to write any software that calls these endpoints.